_notices/rgn0023.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: notice
+parent: RAPIDS General Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rgn
+# Update meta-data for notice
+notice_id: 23 # should match notice number
+notice_pin: false # set to true to pin to notice page
+title: "New 'Try RAPIDS Now' Capability via SageMaker Studio Labs"
+notice_author: RAPIDS Ops
+notice_status: Completed
+notice_status_color: green
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Announcement
+notice_rapids_version: "v22.02"
+notice_created: 2022-02-17
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2022-02-17
+---
+
+## Overview
+
+SageMaker Studio Labs has been added as a new "Try it Now" capability for all Legacy (`v21.12`), Stable (`v22.02`), 
```

Comment:
Lets list this in order Stable, Nightly, and Legacy--no sense in leading with the old stuff!

Reply:
done